lighthouse-core/report/html/report-styles.css

```diff
@@ -1368,6 +1368,10 @@
   text-align: left;
 }
 
+.lh-table-column--code {
+  width: 80%;
```

Comment:
would a much narrower `min-width: 50px` or something also work here? 

In your screenshot both content *and* error are `column--code` and this `width: 80%` ends up falling back to some other table sizing heuristics. There are many other audits where we would not want the table dominated by the code column though.

Reply:
Ah this is what happens when you edit a single element rather than the class name itself. My bad! Yes that's a better solution. So `min-width: 50px` still looks pretty unreadable to me. But 100px works:

**50px**:
![Screenshot 2020-09-01 at 18 42 15](https://user-images.githubusercontent.com/1223960/91887787-24c54380-ec83-11ea-95b6-c97040ec7c69.png)

**100px**:
![Screenshot 2020-09-01 at 18 41 58](https://user-images.githubusercontent.com/1223960/91887825-33abf600-ec83-11ea-9523-8257c515303a.png)

Happy to update accordingly.

Reply:
thanks 100px SGTM 👍 :)

Reply:
Done 👍